### PR TITLE
fix(cli): polish user interaction by switching libraries

### LINKED ISSUE #788: CLI: Deleting inputs on init does not work
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

- CDKTF 0.4.1

### Expected Behavior

If I mistype the project name / description and press the delete key I undo the last character

### Actual Behavior

Escape characters get added

### References

* #0000

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Fixes #788

### CODE REVIEW

File: packages/cdktf-cli/bin/cmds/helper/init.ts [hunk 1]
```diff
@@ -245,33 +245,31 @@ async function gatherInfo(
...
-      "A simple getting started project for cdktf.";
-    projectDescription = readlineSync.question(
-      chalkColour`{greenBright Project Description:} (default: '${projectDescriptionDefault}') `,
-      { defaultInput: projectDescriptionDefault }
-    );
-  }
+  const currentDirectory = path.basename(process.cwd());
+  const projectDescriptionDefault =
+    "A simple getting started project for cdktf.";
+  const questions = [
```
Comment (ansgarm) [inline packages/cdktf-cli/bin/cmds/helper/init.ts:251]: nit: I personally would find it more readable to create an empty array and do `if(!x) arr.push({name: "x", ...})`

File: packages/cdktf-cli/bin/cmds/helper/init.ts [hunk 2]
```diff
@@ -323,23 +325,26 @@ async function getTemplate(templateName: string): Promise<Template> {
...
-      );
-      if (templateName == "") {
-        console.log("No URL was given (received empty string). Aborted.");
-        process.exit(1);
-      }
+    const { template: selection } = await inquirer.prompt([
+      {
+        type: "list",
+        name: "template",
+        message: "What template you want to use?",
```
Comment (ansgarm) [inline packages/cdktf-cli/bin/cmds/helper/init.ts:332]: not your change but, "What template *do you want to use?"

File: packages/cdktf-cli/bin/cmds/helper/init.ts [hunk 3]
```diff
@@ -323,23 +325,26 @@ async function getTemplate(templateName: string): Promise<Template> {
...
-      process.exit(0);
-    }
-    if (selection === options.indexOf(templateOptionRemote)) {
-      templateName = readlineSync.question(
-        "Please enter an URL pointing to the template zip file you want to use: "
-      );
-      if (templateName == "") {
-        console.log("No URL was given (received empty string). Aborted.");
-        process.exit(1);
-      }
```
Comment (ansgarm) [inline packages/cdktf-cli/bin/cmds/helper/init.ts:340]: Was this error handling removed on purpose?

**Review by ansgarm — COMMENTED** [review]
Love the new lib 👍 Only a few small things

File: packages/cdktf-cli/bin/cmds/helper/init.ts (hunk 1 above)
Comment (DanielMSchmidt) [reply packages/cdktf-cli/bin/cmds/helper/init.ts:251]: Yeah that's fair, I'll change it

File: packages/cdktf-cli/bin/cmds/helper/init.ts (hunk 3 above)
Comment (DanielMSchmidt) [reply packages/cdktf-cli/bin/cmds/helper/init.ts:340]: Yes and no, I forgot to add the validation function :D

**Review by ansgarm — APPROVED** [review]

**Pull request merged**